src/FakeItEasy/Configuration/PropertySetterConfiguration.cs

```diff
@@ -6,7 +6,9 @@ namespace FakeItEasy.Configuration
     using FakeItEasy.Core;
     using FakeItEasy.Expressions;
 
-    internal class PropertySetterConfiguration<TValue>
+#pragma warning disable SA1601 // Partial elements should be documented
+    internal partial class PropertySetterConfiguration<TValue>
+#pragma warning restore SA1601 // Partial elements should be documented

```

Comment:
See https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3009. If you prefer, we can globally disable SA1601, or add a StyleCop settings file to ignore it on non public members.

Reply:
Nice new issue. I'm happy with any of the solutions you showed or mentioned.

Reply:
We already disabled [SA1600](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1600.md) globally, so I guess it would make sense to do the same for SA1601. Less clutter in the code that way.